Makefile

```diff
@@ -680,6 +680,10 @@ TAR		:= tar
 UNZIP		:= unzip -qq -u
 GIT		:= git
 WGET		:= wget
+WGET_SHOW_PROGRESS_FLAG := --show-progress
+ifneq (,$(findstring Wget2,$(shell $(WGET) --version)))
```

Comment:
This test is a very good idea. However, I like less that we introduce another `WGET` variable for handling standard flags. This makes its usage a bit more complicated. Can we instead define with `WGET` to do the progress bar? This would be more inline with the other tools (!= compilers/linkers).

```Makefile
ifneq (,$(findstring Wget2,$(shell $(WGET) --version)))
WGET := wget -q --show-progress --progress=bar
else
WGET := wget -q --force-progress --progress=bar
endif
```
Of course, you would shrink the command line at the places where `wget` is called.

Reply:
Good idea, I updated the commit.

Reply:
Thanks a lot for the update :+1: